index.html

```diff
@@ -268,6 +268,74 @@ <h4 class="experience-title">Skills</h4>
         </li>
       </ul>
     </section>
+    <section class="contact-form">
+      <p>
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </p>
+      <form  class="form" action="https://formspree.io/f/xknkyppy" method="POST">
+        <input
+          type="text"
+          name="name"
+          maxlength="30"
+          placeholder="Full name"
+          required
+        />
+        <input type="email" name="email" placeholder="Email address" required />
+        <textarea maxlength="500" name="message" required>
+Enter text here</textarea
+        >
+        <button type="submit">Get in touch</button>
+      </form>
```

Comment:
- [ ] The Get in touch button should follow the button styling: Active, Hover and Pressed (see design)

Reply:
fixed